frontend/src/common/mock/authCodeVerify/authCodeVerify.ts

```diff
@@ -0,0 +1,30 @@
+import { http, HttpResponse } from 'msw';
+
+import { API_ENDPOINTS } from '../../constants/apiEndpoints';
+
+interface AuthCodeVerifyBody {
+  code: string;
+}
+
+const BASE_URL = process.env.BASE_URL;
+
+const DUMMY_AUTH_CODE = '123456';
+
+const postAuthCodeVerify = http.post(
+  `${BASE_URL}${API_ENDPOINTS.AUTH_CODE_VERIFY}`,
+  async ({ request }) => {
+    const body = await request.json();
+
+    if (!body) {
+      return;
```

Comment:
p5: msw는 실제 서버처럼 동작한다고 생각해서 아무것도 return 하지 않기 보다는 에러 코드를 갖는 httpResponse를 반환하는 건 어떨까? 🤔

ex)
```suggestion
      return new HttpResponse(null, { status: 404 });
```

- [msw 공식 문서](https://mswjs.io/docs/http/mocking-responses/error-responses)

Reply:
[refactor: body없는 경우 400 응답 주도록 모킹 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/274/commits/4c01a1bb1b7d937c273a823839bc34bce7818272)